rust/opendp/src/core.rs

```diff
@@ -461,6 +461,38 @@ impl<D: 'static + Domain, M: 'static + Metric> MetricGlue<D, M> {
 // CHAINING & COMPOSITION
 pub struct ChainMT;
 
+impl ChainMT {
+    pub fn make_chain_mt_glue<DI, DX, DO, MI, MX, MO>(
```

Comment:
Perhaps we call these `make_with_glue`? Then they are called like `ChainMT::chain_with_glue(...)` instead of ChainMT::make_chain_mt_glue(...)`.